stage1/tasks/hangman/README.md

```diff
@@ -98,6 +99,15 @@ _It is recommended to print the secret word in the browser's console to facilita
 - [ ] The modal window includes the message about the game's outcome (winning or losing), the secret word and the 'play again' button: `+10`
 - [ ] When the user clicks the 'play again' button, the game starts over by showing a new question and resetting the gallows, the incorrect guesses counter and the underscores for the secret word: `+15`
 
+## Penalties:
+- The question-answer pair remains the same upon refreshing the page _(please, refresh the page **multiple times** to verify this requirement)_: `-20`
+- When clicking on 'play again' button, the newly selected question-answer pair is identical to the previous one: `-20`
```

Comment:
Does this mean the given word has to be removed from the bank somehow? If we have 10 pairs in the bank, it's unlikely, but possible that the pair can be the same next game, so the penalty will apply.

Reply:
@stardustmeg when you choose a new random question, you can check whether it's the same as the previous one and, if yes, randomize again.

Reply:
@elukashova I see your point! Thanks so much for the answer!